components/email/src/Email.svelte

```diff
@@ -274,7 +274,7 @@
       // get body for last message in open thread
       if (thread?.messages?.length > 0) {
         const lastMsgIndex = thread.messages.length - 1;
```

Comment:
`fetchIndividualMessage` is updated to be consistent with the same function in Mailbox component, and better suits its name. 

See the function itself below for more detail.